### ISSUE TITLE
"ReferenceError: Alert is not defined"

### ISSUE THREAD

**Issue by atomantic** [keymetrics/pmx#40]
```
2015-08-03 11:29:22: App name:did-ui-bundler id:1 online
/data/apps/prod/did-ui-bundler/bin/pm2/node_modules/pmx/lib/utils/alert.js:23
Alert = function(opts) {
      ^
ReferenceError: Alert is not defined
    at Object.<anonymous> (/data/apps/prod/did-ui-bundler/bin/pm2/node_modules/pmx/lib/utils/alert.js:23:7)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/data/apps/prod/did-ui-bundler/bin/pm2/node_modules/pmx/lib/Probe.js:5:16)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
2015-08-03 11:29:22: App name:did-ui-bundler id:1 exited
2015-08-03 11:29:22: Script /data/apps/prod/did-ui-bundler/index.js had too many unstable restarts (15). Stopped. "errored"
```

**Comment by Unitech** [comment]
Are you using something like 'use strict'?

**Comment by Unitech** [comment]
fixed in 0.3.27

**Comment by MohanMypeegu** [comment]
I am getting the same issue, Unitech, can you please response to it.
